src/cycax/cycad/features.py

```diff
@@ -141,13 +141,20 @@ def swap_yz(self, rot: float, rotmax: list) -> list:
 nut_specifications: dict[
     str, dict[str, float]
 ] = {  # This is a global variable that will be used to cut the nuts by the OpenSCAD engine.
-    "M3": {
+    "MISO3": {
```

Comment:
M3ISO or ISOM3. I like the first since the 3 is an automatic separator. Else one reads "isom 3".